src/components/BookCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { VscHeart } from "react-icons/vsc";
+import ReactStars from "react-rating-stars-component";
+
+function BookCard() {
+  return (
+    <div className="bg-Cards rounded overflow-hidden shadow-md w-full md:w-72 p-3">
```

Comment:
It could be better to add `font-sans` to make the font styles Montserrat. When you do this the `Book Title` and `$9.99` will become very bold. You can change them to be `font-semibold`

Reply:
i thought it was Montserrat by default, i will change it thanks